net.sourceforge.fuse_emulator.Fuse.yaml

```
@@ -0,0 +1,123 @@
+app-id: net.sourceforge.fuse_emulator.Fuse
+branch: stable
+runtime: org.freedesktop.Platform
+runtime-version: '24.08'
+sdk: org.freedesktop.Sdk
+command: fuse
+rename-desktop-file: fuse.desktop
+rename-icon: fuse
+copy-icon: true
+finish-args:
+  # X11 + XShm access
+  - --share=ipc
+  - --socket=x11
+  # Sound
+  - --socket=pulseaudio
+  - --device=all
+  # Network access (for Spectranet)
+  - --share=network
+  # Home directory (for loading and saving Spectrum software)
+  - --filesystem=home
+build-options:
+  cflags: -O2 -g
+  cxxflags: -O2 -g
+  cppflags: -DG_DISABLE_CAST_CHECKS -DGLIB_DISABLE_DEPRECATION_WARNINGS -DGDK_DISABLE_DEPRECATION_WARNINGS
+  env:
+    V: '1'
+cleanup:
+  - /bin/sdl-config
+  - /include
+  - /lib/*.a
+  - /lib/*.la
+  - /lib/*.so
+  - /lib/pkgconfig
+  - /share/aclocal
+  - /share/applications/mimeinfo.cache
+  - /share/icons/hicolor/icon-theme.cache
+  - /share/man
+modules:
+  # SDL 1.2 compatibility layer for SDL 2
+  - shared-modules/SDL/sdl12-compat.json
+  # Audiofile is used for loading Spectrum data from WAV files
+  - name: libaudiofile
+    config-opts:
+      - --disable-docs
+      - --disable-examples
+      - --disable-flac
+      - --disable-static
+    sources:
+      - type: archive
+        url: http://audiofile.68k.org/audiofile-0.3.6.tar.gz
+        sha256: cdc60df19ab08bfe55344395739bb08f50fc15c92da3962fac334d3bff116965
+      - type: patch
+        path: audiofile-build.patch
+    cleanup:
+      - /bin
+  # Library to handle Spectrum snapshots, tapes and other files
+  - name: libspectrum
+    sources:
+      - type: archive
+        url: https://netcologne.dl.sourceforge.net/project/fuse-emulator/libspectrum/1.5.0/libspectrum-1.5.0.tar.gz
```

please use generic downloads link for sourceforge.

```suggestion
        url: https://download.sourceforge.net/project/fuse-emulator/libspectrum/1.5.0/libspectrum-1.5.0.tar.gz
```